temba/channels/models.py

```diff
@@ -2053,16 +2068,26 @@ def send_twilio_message(cls, channel, msg, text):
         from temba.orgs.models import ACCOUNT_SID, ACCOUNT_TOKEN
 
         callback_url = Channel.build_twilio_callback_url(msg.id)
-        client = TwilioRestClient(channel.org_config[ACCOUNT_SID], channel.org_config[ACCOUNT_TOKEN])
         start = time.time()
 
-        if channel.channel_type == TWILIO_MESSAGING_SERVICE:
-            messaging_service_sid = channel.config['messaging_service_sid']
-            client.messages.create(to=msg.urn_path,
-                                   messaging_service_sid=messaging_service_sid,
-                                   body=text,
-                                   status_callback=callback_url)
+        if channel.channel_type == TWILIO_MESSAGING_SERVICE and not channel.channel_type == TWIML_API:
```

Comment:
This if statement doesn't make sense.. or rather the not clause doesn't do anything as far as I can tell.